### PR TITLE
Checkbox Component

### PR DESCRIPTION
**Pull request by serling**

### CODE REVIEW

File: components/checkbox/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+### Author
+
+Thomas Christoffersen
+
+### Dependencies
+
+- `prop-types`
+- `classnames`
+- `@creuna/lib Icon`
```
Comment (asbjornh) [inline components/checkbox/README.md]: This looks like the name of an NPM package. `@creuna/lib` does not exist and we aren't using npm for distributing these, so another description would be good. You could maybe add a link?

```
- [Icon component](../Icon)

File: components/checkbox/checkbox.jsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      {url ? (
+        <React.Fragment />
+      ) : (
+        <input
+          className="checkbox__input"
+          type="checkbox"
+          name={inputName}
+          onChange={e => onChange(e, id)}
+          id={id}
+          checked={isActive}
```
Comment (asbjornh) [inline components/checkbox/checkbox.jsx]: It seems like this props should be called `checked`

File: components/checkbox/checkbox.jsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cn from "classnames";
+
+import Icon from "../icon";
+
+const themes = {
+  default: "default",
```
Comment (asbjornh) [inline components/checkbox/checkbox.jsx]: The themes have no styling. I think they should be removed

File: components/checkbox/README.md (hunk 1 above)
Comment (serling) [reply components/checkbox/README.md]: Yes, was wondering about this! Thanks.

File: components/checkbox/checkbox.jsx (hunk 2 above)
Comment (serling) [reply components/checkbox/checkbox.jsx]: Agreed.

**Pull request merged**